### DISCUSSION TITLE
Stomach pH on Fed state

### DISCUSSION THREAD

**Discussion by fredsevmartins** [Open-Systems-Pharmacology/Forum#761] [Questions & Problems]
Hi
How can I access the stomach pH on the Fed state? I only can see fasted state. 

thanks
Frederico

![image](https://user-images.githubusercontent.com/45317633/84691198-0719f500-af44-11ea-9ee2-4ecaa75b4b46.png)

**Comment by jzq90** [comment]
This can be set in the Mobi. It is an advanced parameter, so I think it isn't shown in PK-Sim.
Also, the pH after meal is a function of time
![图片](https://user-images.githubusercontent.com/58142266/84868076-b8d53500-b0ae-11ea-8269-3c98030e104c.png)
![图片](https://user-images.githubusercontent.com/58142266/84866866-e8833d80-b0ac-11ea-9d78-16fed6db6dc4.png)
